Build problem after migrating from 1.6 to 2.0

After migration process, I get an error on build. I have to run a second time the build  

1°/ .build/venv/bin/alembic --config alembic.ini upgrade head
2°/ .build/venv/bin/alembic --config alembic_static.ini upgrade head
3°/ .build/venv/bin/themev1tov2
4°/ .build/venv/bin/l10nv1tov2 fr cartoriviera/static/js/Proj/Lang/fr.js cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po
5°/ make -f v2.mk clean
6°/ make -f v2.mk build

-> 

sed -i '/^"POT-Creation-Date: /d' cartoriviera/locale/cartoriviera-client.pot
sed -i '/^"PO-Revision-Date: /d' cartoriviera/locale/cartoriviera-client.pot
mkdir -p cartoriviera/locale/fr/LC_MESSAGES/
[ -f cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po ] || msginit --no-translator --input=cartoriviera/locale/cartoriviera-client.pot --output-file=cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po -l fr
touch --no-create cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po
msgmerge --sort-by-file --backup=none --update cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po cartoriviera/locale/cartoriviera-client.pot
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "coordinates" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697:32: syntax error
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "http" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "www" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "cartoriviera" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "ch" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "mobile" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "streetview" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "php" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "x" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "x" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "y" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "y" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "_blank" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "photos_historiques" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "liste_photos" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "php" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "coordx" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "x" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "coordy" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "y" unknown
cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po:1697: keyword "_blank" unknown
msgmerge: found 22 fatal errors
CONST_Makefile:664: recipe for target 'cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po' failed
make: *** [cartoriviera/locale/fr/LC_MESSAGES/cartoriviera-client.po] Error 1

7°/ make -f v2.mk build -> OK

--- a/c2cgeoportal/scripts/l10nv1tov2.py
+++ b/c2cgeoportal/scripts/l10nv1tov2.py
@@ -93,5 +93,5 @@ msgstr ""
                 destionation.write(("""
 msgid "%s"
 msgstr "%s"
-""" % (key, value)).encode("utf-8"),
+""" % (key, value.replace('"', '\\"'))).encode("utf-8"),
                 )

